.github/workflows/delete-review-app.yml

```diff
@@ -1,131 +1,77 @@
-name: Delete Review App
+name: Delete review app on AKS
 
 on:
   pull_request:
-    types: [closed]
-    branches: [main]
-  workflow_dispatch:
-    inputs:
-      pr_number:
-        description: PR number of review app to delete
-        required: true
-        type: string
+    branches:
+      - main
+    types:
+      - closed
+
 jobs:
   delete-review-app:
```

Comment:
if this job is for existing non-aks review apps I don't think it will work. Do we need to keep the old review apps? Maybe check with devs.

Reply:
Leaving this as suggested by Colin for now, i guess we'll have a ticket in the future for clean up and finalising the migration

Reply:
Why delete the old code? It's required to delete the old review apps

Reply:
For the last few paas migrations we started removing the paas review apps as we created the aks ones, and moved across any existing. So just a question if we were do the same here. I didn't see the point of testing non-aks build/delete in this pr until that was answered.